lib/ansible/modules/cloud/google/gcp_healthcheck.py

```diff
@@ -0,0 +1,464 @@
+#!/usr/bin/python
+# Copyright 2017 Google Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: gcp_healthcheck
+version_added: "2.4"
+short_description: Create, Update or Destroy a Healthcheck.
+description:
+    - Create, Update or Destroy a Healthcheck. Currently only HTTP and
+      HTTPS Healthchecks are supported. Healthchecks are used to monitor
+      individual instances, managed instance groups and/or backend
+      services. Healtchecks are reusable.
+    - Visit
+      U(https://cloud.google.com/compute/docs/load-balancing/health-checks)
+      for an overview of Healthchecks on GCP.
+    - See
+      U(https://cloud.google.com/compute/docs/reference/latest/httpHealthChecks) for
+      API details on HTTP Healthchecks.
+    - See
+      U(https://cloud.google.com/compute/docs/reference/latest/httpsHealthChecks)
+      for more details on the HTTPS Healtcheck API.
+requirements:
+  - "python >= 2.6"
+  - "google-api-python-client >= 1.6.2"
+  - "google-auth >= 0.9.0"
+  - "google-auth-httplib2 >= 0.0.2"
+notes:
+  - Only supports HTTP and HTTPS Healthchecks currently.
+author:
+  - "Tom Melendez (@supertom) <tom@supertom.com>"
+options:
+  check_interval:
+    description:
+       - How often (in seconds) to send a health check. The default
+         value is 5 seconds.
```

Comment:
Please use `default:` and remove from `description:`

http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#documentation-block